src/openforms/prefill/contrib/objects_api/apps.py

```diff
@@ -0,0 +1,12 @@
+from django.apps import AppConfig
+from django.utils.translation import gettext_lazy as _
+
+
+class ObjectsApiApp(AppConfig):
+    name = "openforms.prefill.contrib.objects_api"
+    label = "objects_api"
```

Comment:
```suggestion
    label = "prefill_objects_api"
```
please namespace django app labels, as they must be unique across the project